src/common/android/packageNameResolver.ts

```diff
@@ -54,10 +54,10 @@ export class PackageNameResolver {
     }
 
     /**
-     * Gets the default package name, based on the applicaiton name.
+     * Gets the default package name, based on the application name.
      */
     private getDefaultPackageName(applicationName: string): string {
-        return PackageNameResolver.DefaultPackagePrefix + applicationName;
+        return (PackageNameResolver.DefaultPackagePrefix + applicationName).toLowerCase();
```

Comment:
I know... It's just to be extra sure in case someone changes the other variable to something uppercase...